tests/tests/tier0/monitor-open-close/python/monitor.py

```diff
@@ -0,0 +1,25 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+from dasbus.error import DBusError
+
+from bluechi.api import Manager, Monitor
+
+# Subscribing to a closed monitor should result in a DBusError since the monitor has removed.
+# In this test we first subscribe to an open monitor to verify everything works, then close
+# the monitor and expect an DBusError when subscribing on a closed monitor.
+
+
+def main():
+    mgr = Manager()
+    monitor_path = mgr.create_monitor()
+    monitor = Monitor(monitor_path=monitor_path)
+    monitor.subscribe('not-important-node', 'not-important-unit')
+    monitor.close()
+    try:
+        monitor.subscribe('not-important-node-2', 'not-important-unit-2')
+    except DBusError:
+        pass
```

Comment:
we should do an "assert" on the error here, to check that we got the right error

Reply:
Done :heavy_check_mark: The assert should also provide a better message when it fails and help debugging issues.